.github/workflows/sonar.yml

```diff
@@ -1,18 +1,11 @@
 name: Sonar
 on:
-  workflow_call:
-    secrets:
-      SONAR_TOKEN:
-        required: true
-#  pull_request:
-#    branches:
-#      - develop
-#    types:
-#      - opened
-#      - reopened
-#      - synchronize
-#      - labeled
-#      - unlabeled
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
```

Comment:
hi there :wave: the PR description is not very extensive, so I'm lacking a bit of context for the review.
Do we intend to run it only on `develop`?

Reply:
Hi Mariona, as it is now, it will run in both develop and in every PR branch against develop. So yes, you need to specify to SonarCloud what is the repo default branch, in our case is develop.

Reply:
thank you!! clear now :+1: